src/exporter/exporter/csv.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 const (
-	adcVref = 33.0
+	adcVref            = 33.0
+	microSecondsInHour = 60.0 * 60.0 * 1000000.0
```

Comment:
we could use instead the duration constants from the standard `time` lib: `time.Hour` will give you duration in nanoseconds, so `float64(time.Hour / 1000)` will be in microseconds